LeaderboardBackend/Controllers/AccountController.cs

```diff
@@ -167,4 +166,32 @@ public async Task<ActionResult<LoginResponse>> Login([FromBody] LoginRequest req
             notFound => Unauthorized()
         );
     }
+
+    /// <summary>
+    ///     Confirms a user account.
+    /// </summary>
+    /// <param name="id">The confirmation token.</param>
+    /// <param name="confirmationService">IAccountConfirmationService dependency.</param>
+    /// <response code="200">The account was confirmed successfully.</response>
+    /// <response code="400">The token was malformed.</response>
```

Comment:
Our Guid model binder should trigger this, right?

Reply:
I believe so but I can put an additional test in if you want.

Reply:
Yes, please. We don't have a test that checks for this currently as well.

Reply:
Actually, 404 would be more appropriate here, since the resource we're trying to locate could not be found. From the client's perspective, the format of the token is an implementation detail and does not matter.

Reply:
In that case, you could remove that line then